### PR TITLE
added two posts

### PR DESCRIPTION
**Pull request by kidfribble**

### CODE REVIEW

**Comment by kidfribble** [comment]
I'm going to merge and publish one of the posts tomorrow, then one on Wednesday.

File: source/posts/2015-01-13-book-design-and-the-web.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+twitter: "rgbcolor"
+social: true
+summary: "Really understand your content, then select its format."
+published: false
+tags: design, observations
+
+---
+
+
+#Book design and the web
```
Comment (duggiefresh) [inline source/posts/2015-01-13-book-design-and-the-web.md]: I believe GitHub's markdown requires a space between the `#` and `Book`.

File: source/posts/2015-01-13-book-design-and-the-web.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+![](/images/post-assets/maria/blog-books-and-web-design-03.jpg)
+
+The final section holds student photographs. They are shown in an order determined by the value of the photo: from dark to light, avoiding forced storylines between photographs that do not necessarily relate to each other. The paper selection has changed once again, and the photos are given full prominence with minimal captions.
+![](/images/post-assets/maria/blog-books-and-web-design-04.jpg)
+
+I find this book to be a great example of logical and thoughtful design: content is well supported and enhanced through the technology of book production. Everything, from the binding, to the paper, to the colors and page layouts, supports the content of the book. I also think it’s a strong parallel to the use of technology to support users, and the things they want to do, when we build products on the web.
+
+##Formats on the web
```
Comment (duggiefresh) [inline source/posts/2015-01-13-book-design-and-the-web.md]: Space between `##` and `Formats`

File: source/posts/2015-01-13-book-design-and-the-web.md (hunk 1 above)
Comment (duggiefresh) [inline source/posts/2015-01-13-book-design-and-the-web.md]: Hmm... Nevermind, it seems that it doesn't matter! :)

File: source/posts/2015-01-13-book-design-and-the-web.md (hunk 2 above)
Comment (duggiefresh) [inline source/posts/2015-01-13-book-design-and-the-web.md]: Doesn't matter.

File: source/posts/2015-01-13-book-design-and-the-web.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+While technology allows us to make things respond in a specific manner to an action, to show only the necessary things to the right person, or to add delightful and playful details to already common tasks, it is the thoughtful and often economical selection of the “formats” we use on the web that makes for a strong, impactful project.
+
+For example, using Ember.js - [a technology that allows us to change portions of a web page without reloading the entire thing](http://www.google.com/url?q=http%3A%2F%2Freefpoints.dockyard.com%2F2015%2F01%2F07%2Fcomplex-search-pages-feel-better-in-ember.html) - allows us to enhance a search page very effectively. We could use the same technology to enhance a gallery type page (whose primary purpose is to show static content) but the improvement would be marginal and not cost effective. The [Inexhibit platform](http://www.indexhibit.org/what-who-why-how/) focuses on *“a community of users who place emphasis on content over complicated website design”* and offers a minimal tool for artists to create a gallery and show their work.
```
Comment (duggiefresh) [inline source/posts/2015-01-13-book-design-and-the-web.md]: Should the url to the Reefpoints blog post be: `http://reefpoints.dockyard.com/2015/01/07/complex-search-pages-feel-better-in-ember.html`?

**Comment by duggiefresh** [comment]
Both of these posts are great! Love the ending to the bathroom one.

File: source/posts/2015-01-13-book-design-and-the-web.md (hunk 3 above)
Comment (kidfribble) [inline source/posts/2015-01-13-book-design-and-the-web.md]: Good catch, yes it should

**Pull request merged**